common/cert.c

```diff
@@ -2,12 +2,17 @@
 // Licensed under the MIT License.
 
 #include <openenclave/bits/safecrt.h>
-#include <openenclave/internal/crypto/asn1.h>
 #include <openenclave/internal/crypto/cert.h>
 #include <openenclave/internal/raise.h>
 #include <openenclave/internal/utils.h>
 #include "common.h"
 
+/* oe_get_crl_distribution_points is shared between host OpenSSL and
```

Comment:
Is this applicable for TrustZone?

Reply:
We'll need to clarify with Hernan, but common/cert.c is currently used for shared code for SGX reporting between host and enclave. There needs to be a larger refactor that I would scope out of this PR if ARM needs to reuse this code.